src/scss/components/_button.scss

```diff
@@ -133,6 +130,14 @@
     }
   }
 
+  &--full-card {
+    border-top: get-border(default);
+    border-bottom-right-radius: $borderRadius;
+    border-bottom-left-radius: $borderRadius;
```

Comment:
Now we have two border radius. What do you think of making clear what the $borderRadius does? Maybe you changed his name?